codegen/src/main/resources/software/amazon/awssdk/codegen/waiters/WaitersRuntime.java

```diff
@@ -158,13 +146,20 @@ private static Value newProjection(Collection<?> values) {
          */
         public Object value() {
             switch (type) {
-                case NULL: return null;
-                case POJO: return pojoValue;
-                case INTEGER: return integerValue;
-                case STRING: return stringValue;
-                case BOOLEAN: return booleanValue;
-                case LIST: return listValue;
-                default: throw new IllegalStateException();
+                case NULL:
+                    return null;
+                case POJO:
+                    return pojoValue;
+                case INTEGER:
+                    return integerValue;
+                case STRING:
+                    return stringValue;
+                case BOOLEAN:
+                    return booleanValue;
+                case LIST:
+                    return listValue;
+                default:
+                    throw new IllegalStateException();
```

Comment:
You monster.

Reply:
Sorry! haha, I didn't even notice until after I opened the PR. It was Intellij's autoformatting I swear.